introduction/what-is.md

```diff
@@ -0,0 +1,48 @@
+That is to say team A implements the feature directly in team B's software and submits a pull request with the code changes.
+Team B partners to review and accept the submitted code.
+
+In this example, we call team A the **Guest** team and team B the **Host** team.
+The terms **Guest** and **Host** suggest a situation analogous to receiving a visitor in the home.
+In that situation, most people want to be a good host.
+They ensure that things are kept neat and tidy in anticipation of their guests' arrival.
+Visitors are greeted at the door and invited in.
+They can use the features and utilities that are in the home's public areas.
+There may be a few house rules that guests are asked to follow.
+Similarly, most guests want to show respect for the home and their host.
+They're careful with the items in the house and follow the rules for the duration of their stay.
+They may hope for or expect a return invitation as long as they've been courteous and polite.
+These concepts around a home visit are a metaphor for the attitude and behaviors that teams should bring as one hosts another making a guest contribution to the codebase.
+
+Let's take a closer look at how the mechanics of the inner source process can work.
+To aid in this explanation, we'll name a few key individuals on the guest and host teams.
+First, the **Product Owner** determines what functionality the host team is willing to accept as a contribution.
+The **Contributor** is the individual on the guest team that submits the code contribution for review by the host team.
+The **Trusted Committer** represents the host team in providing any timely support and mentorship that the contributor needs to successfully submit the pull request.
+
+With those definitions, here is the basic outline of an inner source contribution.
+
+* Guest team or contributor requests a feature from the host team.
+* Product owner ensures that user stories representing the feature request are created.
+These stories should describe the requested feature in terms agreeable to the guest team.
+They also list any details from the host team on how the feature should be delivered in order for the work to be accepted.
+Examples of such details include architecture constraints, coding conventions, dependency usages, data contracts, etc.
+* Supported by the trusted committer, the contributor submits the pull request to implement the requested feature.
+
+Note that these steps do not assume a specific system for the general organization of a team's time or priorities. Inner source assumes that teams already have existing methods of organization and provides a framework of how to use them to work together where there is a guest team desiring to contribute code to a host.
+
+This option works well for the guest team because they get the functionality they need when they need it without taking on the long-term burden of maintenance of the solution.
```

Comment:
```suggestion
## Realized advantages
This option works well for the guest team because they get the functionality they need when they need it without taking on the long-term burden of maintenance of the solution.
```